include/glow/Importer/ONNX.h

```diff
@@ -49,6 +49,12 @@ class ONNXModelLoader
   /// file.
   bool loadProtoFile(onnx::GraphProto &net, const std::string &filename);
 
+  /// ONNX model ir_version;
+  size_t ir_version;
```

Comment:
Any plans on using this? If not, then I'd prefer to keep is a local var where it's extracted.

Reply:
For ONNX "constant", if ir_version < 3, the "type" is not required. But if ir_version is >=3, the "type" is required. Since so far we don't have such  a model, I didn't modify this part.

Reply:
Please follow the variable naming conventions. irVersion_